pysignalscope/scope.py

```diff
@@ -137,8 +164,18 @@ def modify(self, channel_data_factor: Optional[float] = None, channel_data_offse
                     index_list_to_remove.append(count)
             self.channel_time = np.delete(self.channel_time, index_list_to_remove)
             self.channel_data = np.delete(self.channel_data, index_list_to_remove)
+            modify_flag = True
+
         if channel_linestyle is not None:
             self.channel_linestyle = channel_linestyle
+            modify_flag = True
+        pass
```

Comment:
why a single `pass`?

Reply:
In special cases a missing 'pass' leads to problems. But this is no special case. 
Conclusion: Pass will be removed.